cpp/daal/src/algorithms/dtrees/forest/oneapi/df_tree_level_build_helper_oneapi.h

```diff
@@ -68,19 +68,20 @@ class TreeLevelBuildHelperOneAPI
                                           services::internal::sycl::UniversalBuffer & totalSum, size_t localSize, size_t nSubgroupSums);
     services::Status fillOOBRowsListByBlocks(const services::internal::sycl::UniversalBuffer & rowsBuffer, size_t nRows,
                                              const services::internal::sycl::UniversalBuffer & partialPrefixSums,
-                                             services::internal::sycl::UniversalBuffer & oobRowsList, size_t localSize, size_t nSubgroupSums);
+                                             services::internal::sycl::UniversalBuffer & oobRowsList, size_t localSize, size_t nSubgroupSums,
+                                             size_t nOOBRows);
 
     services::Status getOOBRows(const services::internal::sycl::UniversalBuffer & rowsList, size_t nRows, size_t & nOOBRows,
                                 services::internal::sycl::UniversalBuffer & oobRowsList);
```

Comment:
`nOOBRows` in this particular two methods are passed inconsistently - by value and by reference respectively. What is the reason for this?

Reply:
getOOBRows fills oobRowsList and sets nOOBRows. fillOOBRowsListByBlocks is called inside it after nOOBRows was set.